src/main/java/hudson/plugins/git/extensions/impl/PruneStaleTag.java

```diff
@@ -144,8 +161,20 @@ public String toString() {
         return "PruneStaleTag {}";
     }
 
+    @Symbol("pruneTags")
     @Extension
     public static class DescriptorImpl extends GitSCMExtensionDescriptor {
+
+        @Override
+        public GitSCMExtension newInstance(StaplerRequest req, JSONObject formData) throws FormException {
+            if (!formData.containsKey("pruneTags")) {
```

Comment:
I may not be understanding this correctly, since I've never seen a usage like this, but I assume this will make the default for pruneTags to be true.  It should be false to retain behavioral compatibility with previous releases.

Reply:
When jenkins send the job configuration on save the request contains only two parameters.
* starpler
* $class

Since pruneTags is not in the request the build-in mechanism creates a default value for primitive type to true. So to make that field true per default I have add to the formData request like was sent by UI. If the field is sent than I leave the sent value.

There are not other releases and I can not make that false per default otherwise this extension will never work unless in pipeline

Reply:
changed to make it comprensible, I think this extensions will not have futher options to set

Reply:
Just to clarify this do the same of jenkins.plugins.git.traits.PruneStaleTagTrait constructor. So that multibranch pipeline, syntax generator and freejob have the same default value